run_page/garmin_sync_cn_global.py

```diff
@@ -83,3 +94,31 @@
         garmin_global_client.upload_activities_files(to_upload_files, False)
     )
     loop.run_until_complete(future)
+
+    # Save synced activity list for speeding up
+    synced_activity.extend(new_ids)
+    save_synced_activity_list(synced_activity)
+
+    # Step 2:
+    # Generate track from gpx file
+
+    # Todo: use fit file to directly generate track after fixing issue 584
+    # https://github.com/yihong0618/running_page/issues/584
+    folder = GPX_FOLDER
+    if not os.path.exists(folder):
+        os.mkdir(folder)
+    downloaded_ids = get_downloaded_ids(folder)
+
+    loop = asyncio.get_event_loop()
+    future = asyncio.ensure_future(
+        download_new_activities(
+            secret_string_cn,
+            auth_domain,
+            downloaded_ids,
+            is_only_running,
+            folder,
+            "gpx",
+        )
+    )
+    loop.run_until_complete(future)
+    make_activities_file(SQL_FILE, folder, JSON_FILE, file_suffix="gpx")
```

Comment:
After fixing #582, you can delete the above code used for downloading GPX files, as the "FIT" file has already been downloaded before.